pyat/at/tracking/patpass.py

```diff
@@ -93,6 +93,12 @@ def patpass(ring, r_in, nturns=1, refpts=None, losses=False, pool_size=None,
         coordinates at which the particle is lost. Set to zero for particles
         that survived
     """
+    if start_method is not None:
+        try:
+            multiprocessing.set_start_method(start_method)
+        except RuntimeError:
+            pass
+
```

Comment:
Ok, this will work (may be) on the 1st call to `patpass`. Later, the call to `get_start_method` will have frozen the method. You should put a warning mentioning the failure (which may also occur on 1st call since we do not understand the crashes)

Reply:
Yes this is work in progress... I stop for now, more tomorrow, have a nice evening! (Still these crashes are not understood...)